va/va_vpp.h

```diff
@@ -1298,6 +1298,18 @@ typedef struct _VAProcFilterParameterBufferTotalColorCorrection {
     float                             value;
 } VAProcFilterParameterBufferTotalColorCorrection;
 
+/** @name Video Processing Human Vision System (HVS) Denoise Mode.*/
+/**@{*/
+/** \brief Default Mode. This mode is decided in driver to the appropriate mode. */
+#define VA_PROC_HVS_DENOISE_DEFAULT               0
+/** \brief Auto BDRate Mode. */
```

Comment:
hi @FurongZhang , could you help to add more comments for these options? from application view, I dont know how to select it